subsys/bluetooth/services/ots/ots_client.c

```diff
@@ -1185,6 +1291,53 @@ int bt_ots_client_read_object_data(struct bt_ots_client *otc_inst,
 	return oacp_read(conn, inst);
 }
 
+int bt_ots_client_write_object_data(struct bt_ots_client *otc_inst,
+				    struct bt_conn *conn, const void *buf, size_t len,
+				    off_t offset, enum bt_ots_oacp_write_mode mode)
+{
+	struct bt_otc_internal_instance_t *inst;
+
+	CHECKIF(!conn) {
+		BT_WARN("Invalid Connection");
+		return -ENOTCONN;
+	}
+	CHECKIF(!otc_inst) {
```

Comment:
```suggestion
	}
	
	CHECKIF(!otc_inst) {
```

Applies multiples places